qc_tests/humidity.py

```diff
@@ -40,11 +40,11 @@ def get_repeating_dpd_threshold(temperatures: utils.Meteorological_Variable,
 
     # only process further if there are enough locations
     if len(locs) > 1:
-        repeated_string_lengths, _, _ = utils.prepare_data_repeating_string(locs, diff=1, plots=plots, diagnostics=diagnostics)
+        repeated_streak_lengths, _, _ = utils.prepare_data_repeating_streak(locs, diff=1, plots=plots, diagnostics=diagnostics)
```

Comment:
standardised naming so that calling these _streaks_ not _strings_ (which is probably safer giving typing)